analysis/analyze-hotspots/build.gradle

```diff
@@ -36,6 +36,18 @@ dependencies {
     natives "com.esri.arcgisruntime:arcgis-java-resources:$arcgisVersion"
 }
 
+task apiKey {
```

Comment:
In the future this might be a bit more of a generic task, and mostly it creates the properties file, so if you choose to rename the variables inside, maybe we could call it:
```suggestion
task createProperties {
```